.github/actions/mint-token/action.yml

```diff
@@ -38,17 +41,31 @@ runs:
         id_token_audience: '${{ inputs.service_audience }}'
         id_token_include_email: true
 
+    - id: 'gettoken'
```

Comment:
From comment above, move this into the CI workflow as a separate job and then use the resulting token to call the minty endpoint manually. Use the token response to call the github api (this is strange because were in a workflow but /shrug).

Also, can you make sure to use github-script and set the response of the minty call as a secret so we dont leak the token.

Reply:
Moved the test into the CI workflow. I'm not quite sure how to set the response as a secret using github-script, but I think the way I did things with curl wouldn't expose the token?